### PR TITLE
fix #76089 wpshare.net

### LINKED ISSUE #76089: wpshare.net
### Issue URL (Anti Adblock Script)
 [https://wpshare.net/download/?server=aHR0cHM6Ly93d3c1LnppcHB5c2hhcmUuY29tL3YvaWxCN0w5NnovZmlsZS5odG1s](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwpshare.net%2Fdownload%2F%3Fserver%3DaHR0cHM6Ly93d3c1LnppcHB5c2hhcmUuY29tL3YvaWxCN0w5NnovZmlsZS5odG1s)
### Comment
Username: `@adguard1122`
> adguard gets detected on websaite
> --
> The information below is automatically generated. Do not delete it.
> 
> Referrer:
> https://wpshare.net/claue-2-0-9-clean-minimal-elementor-woocommerce-theme/
> 
> Custom userscripts:
> https://greasyfork.org/scripts/735-anti-adblock-killer-reek/code/Anti-Adblock Killer | Reek.user.js
> https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> https://userscripts.adtidy.org/release/adguard-wot/1.0/wot.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Edge 88.0.705.81 (Chromium-based)
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180)
DNS filtering: | server: `https://dns.cloudflare.com/dns-query`<br>
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>Fanboy's Enhanced Tracking List, <br/>Fanboy's Anti-Facebook List, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> Fanboy's Social Blocking List, <br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> EasyList Italy, <br/>Liste AR, <br/>AdBlockID, <br/>Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>Polish Anti Adblock Filters, <br/>AdGuard French, <br/>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Dutch, <br/>AdGuard Spanish/Portuguese
Custom filter subscriptions: | https://raw.github.com/reek/anti-adblock-killer/master/anti-adblock-killer-filters.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#76089

### CODE REVIEW

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -3,6 +3,8 @@
 !
 ! https://github.com/easylist/easylist/pull/6896#pullrequestreview-599945327
 cambb.xxx#?#.thumb div.ad:upward(.thumb)
+! https://github.com/AdguardTeam/AdguardFilters/issues/76089
+wpshare.net#?#section[class^="widget wpshare-widget"] > h4[class^="widget-title"]:contains(Advertisement)
```
Comment (Alex-302) [inline EnglishFilter/sections/css_extended.txt:7]: I don't see it. Only dark leftovers `wpshare.net##div[style^="width: 100%;height: 300px;display: flex;"]`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/css_extended.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt:7]: Ok go to homepage.

Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt:7]: This rule `||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=wpshare.net` hides dark leftovers already.

Comment (Alex-302) [reply EnglishFilter/sections/css_extended.txt:7]: Hm. Sure.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**